src/ontology/mondo-edit.obo

```diff
@@ -574661,6 +574661,14 @@ is_a: MONDO:0022800 {source="https://clinicalgenome.org/affiliation/40065/", sou
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7379" xsd:anyURI
 
+[Term]
+id: MONDO:1040000
```

Comment:
Let's make sure that this ID is not reused. Otherwise, it will create a conflict as soon as the ID is in Master (via one of the PR)

Reply:
Comment addressed.